docs/1.0. Receiver Capabilities.md

```diff
@@ -142,6 +142,14 @@ For example:
 * When a Receiver supports a few options natively, and many that require some transformation, it MAY explicitly associate a positive value just with the native options (since the many will have a lower effective value of 0).
 * When a Receiver supports many options well, but a few low-quality options are provided, perhaps for wider compatibility, it MAY explicitly associate a negative value just with those options (since the many will have a higher effective value of 0).
 
+#### Constraint Set Enabled
+
+The metadata attribute 'urn:x-nmos:cap:meta:enabled' MAY be used to indicate Constraint Sets which do not apply to the current operating configuration of a Receiver, but which can be enabled via out of band configuration.
+
+A Controller MUST NOT take Constraint Sets into consideration when this boolean is set to `false`, unless the Controller has the capability to perform the required out of band configuration. Controllers MAY use this attribute as a hint to users that a Sender and Receiver could be connected subject to a reconfiguration.
```

Comment:
I think it's wise to avoid 'capability' here.
Uncapitalized 'b' in 'boolean' was also bothering me, so I'm changing it. :-)

```suggestion
A Controller MUST NOT take Constraint Sets into consideration when this attribute is set to `false`, unless the Controller is capable of performing the required out of band configuration. Controllers MAY use this attribute as a hint to users that a Sender and Receiver could be connected subject to a reconfiguration.
```